src/corelibs/U2Core/src/gobjects/MultipleChromatogramAlignmentObject.cpp

```diff
@@ -260,6 +260,44 @@ void MultipleChromatogramAlignmentObject::trimRow(const int rowIndex, int curren
     updateCachedMultipleAlignment(modificationInfo);
 }
 
+void MultipleChromatogramAlignmentObject::updateAlternativeMutations(bool showAlternativeMutations, int threshold, U2OpStatus& os) {
+    for (int i = 0; i < getNumRows(); i++) {
+        const MultipleChromatogramAlignmentRow& mcaRow = static_cast<const MultipleChromatogramAlignmentRow&>(getRow(i));
+        qint64 ungappedLength = mcaRow->getUngappedLength();
+
+        QHash<qint64, char> newCharList;
+        for (int j = 0; j < ungappedLength; j++) {
+            bool ok = false;
+            auto res = mcaRow->getTwoHighestPeaks(j, ok);
+            if (!ok) {
+                continue;
+            }
+
+            double minimumThresholdValue = (double)res.second.value / res.first.value * 100;
+            char newChar = 'A';
+            if (minimumThresholdValue < threshold || !showAlternativeMutations) {
```

Comment:
Unused value. You can keep newChar uninitialized or use ? operator instead of the if/else

Reply:
OK, Uninitialized.